j2cl/lit/src/tokens/shell-tokens.css

```
@@ -50,19 +50,23 @@ shell-root {
    * width and the wave-panel gutter can be tuned in one place. The
    * fallbacks preserve V-1's 260-296 px rail when wavy-tokens.css has
    * not been loaded (e.g. signed-out branch / SSR-only smoke). */
-  grid-template-columns: minmax(var(--wavy-rail-width-min, 260px), var(--wavy-rail-width-max, 296px)) 1fr;
+  --j2cl-search-rail-width: var(--wavy-rail-width-max, 296px);
+  grid-template-columns:
+    minmax(var(--wavy-rail-width-min, 260px), var(--j2cl-search-rail-width))
+    6px
+    minmax(0, 1fr);
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)